app/components/form/instance-table.js

```diff
@@ -31,9 +32,7 @@ export default class InstanceTableComponent extends Component {
       `/form-content/${this.formInfo.form.definition.id}/instances/${instanceId}`,
       {
         method: 'DELETE',
-        headers: {
-          'Content-Type': 'application/vnd.api+json',
-        },
+        headers: CONTENT_HEADER,
```

Comment:
JSON_API_CONTENT_HEADER?

Reply:
actually would just put the value for content type in the constant so you can extend headers if you want (without the hassle of cloning etcetc)